services/error.go

```diff
@@ -17,6 +20,10 @@ var (
 	ErrObjectNotExist = errors.New("object not exist")
 	// ErrPermissionDenied means this operation doesn't have enough permission.
 	ErrPermissionDenied = errors.New("permission denied")
+	// ErrObjectNotPart means the object to be operated is not a part object.
+	ErrObjectNotPart = errors.New("object is not a part object")
+	// ErrInvalidListMode means the provided list mode is invalid.
+	ErrInvalidListMode = errors.New("invalid list mode")
```

Comment:
Please use the style like `<noun><adj>`. I prefer `ListModeInvalid` instead of `InvalidListMode`.

We can discuss the error code style in AOS-47.